Gemfile.lock

```diff
@@ -82,80 +82,80 @@ GEM
     crass (1.0.6)
     erubi (1.9.0)
     execjs (2.7.0)
-    ffi (1.12.2)
+    ffi (1.13.1)
     globalid (0.4.2)
       activesupport (>= 4.2.0)
-    i18n (1.8.2)
+    i18n (1.8.5)
       concurrent-ruby (~> 1.0)
     jbuilder (2.10.0)
       activesupport (>= 5.0.0)
-    jquery-rails (4.3.5)
+    jquery-rails (4.4.0)
       rails-dom-testing (>= 1, < 3)
       railties (>= 4.2.0)
       thor (>= 0.14, < 2.0)
     listen (3.1.5)
       rb-fsevent (~> 0.9, >= 0.9.4)
       rb-inotify (~> 0.9, >= 0.9.7)
       ruby_dep (~> 1.2)
-    loofah (2.4.0)
+    loofah (2.6.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
     method_source (1.0.0)
-    mimemagic (0.3.4)
+    mimemagic (0.3.5)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.14.0)
+    minitest (5.14.1)
     msgpack (1.3.3)
     nio4r (2.5.2)
-    nokogiri (1.10.9)
+    nokogiri (1.10.10)
       mini_portile2 (~> 2.4.0)
     popper_js (1.16.0)
-    public_suffix (4.0.3)
+    public_suffix (4.0.5)
     puma (4.3.5)
       nio4r (~> 2.0)
     rack (2.2.3)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
-    rails (6.0.2.2)
-      actioncable (= 6.0.2.2)
-      actionmailbox (= 6.0.2.2)
-      actionmailer (= 6.0.2.2)
-      actionpack (= 6.0.2.2)
-      actiontext (= 6.0.2.2)
-      actionview (= 6.0.2.2)
-      activejob (= 6.0.2.2)
-      activemodel (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
-      activestorage (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    rails (6.0.3.2)
+      actioncable (= 6.0.3.2)
+      actionmailbox (= 6.0.3.2)
+      actionmailer (= 6.0.3.2)
+      actionpack (= 6.0.3.2)
+      actiontext (= 6.0.3.2)
+      actionview (= 6.0.3.2)
+      activejob (= 6.0.3.2)
+      activemodel (= 6.0.3.2)
+      activerecord (= 6.0.3.2)
+      activestorage (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       bundler (>= 1.3.0)
-      railties (= 6.0.2.2)
+      railties (= 6.0.3.2)
       sprockets-rails (>= 2.0.0)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.3.0)
       loofah (~> 2.3)
-    railties (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    railties (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       method_source
       rake (>= 0.8.7)
       thor (>= 0.20.3, < 2.0)
     rake (13.0.1)
-    rb-fsevent (0.10.3)
+    rb-fsevent (0.10.4)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
-    regexp_parser (1.7.0)
+    regexp_parser (1.7.1)
     ruby_dep (1.5.0)
     rubyzip (2.3.0)
     sass-rails (6.0.0)
       sassc-rails (~> 2.1, >= 2.1.1)
-    sassc (2.2.1)
+    sassc (2.4.0)
```

Comment:
### [sassc](https://github.com/sass/sassc-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.2.1 → 2.4.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **36** commits. See the full changes on [the compare page](https://github.com/sass/sassc-ruby/compare/v2.2.1...v2.4.0).

These are the first 10 commits:
- (5de84be) [Set appropriate encoding for source_map.](https://github.com/sass/sassc-ruby/commit/5de84bec0056f4e345259dd6c68c60d1ba990d3d)
- (7e2cdb5) [Bug fix for the commit 5de84be.](https://github.com/sass/sassc-ruby/commit/7e2cdb56b0df3d423fb3d7864c6f016082bf4cc0)
- (d492298) [Default --march-tune-native to false](https://github.com/sass/sassc-ruby/commit/d49229898c5c4e8a142129bdb5043bb01a479812)
- (681bcff) [always dispose data context](https://github.com/sass/sassc-ruby/commit/681bcffe894404d65596be3b03d0548fbe4e1223)
- (5c6c7ae) [allow passing functions directly](https://github.com/sass/sassc-ruby/commit/5c6c7aedec37a506144c1141663e2f200f2864fc)
- (08b80f8) [Update libsass to 3.6.2](https://github.com/sass/sassc-ruby/commit/08b80f83b14e5abcf32ba974968e1b3a1dee08e3)
- (eb02996) [disable lto flag for darwin + nix](https://github.com/sass/sassc-ruby/commit/eb02996ed1033816d6205f6aece5ada8fab71c1d)
- (dc0e251) [Update libsass to 3.6.3](https://github.com/sass/sassc-ruby/commit/dc0e2518b081c42b4e458c3e7fdf2635c24f5323)
- (30daccd) [Test on TruffleRuby in TravisCI](https://github.com/sass/sassc-ruby/commit/30daccd2dba6ffe55ab7a046c1e3461f42f05b44)
- (28aa5c7) [Use RbConfig::MAKEFILE_CONFIG['DLEXT'] instead of hardcoding…](https://github.com/sass/sassc-ruby/commit/28aa5c7102a253f61f34eceb74cf1564442865fb)


</details>